src/server/controller/cycleData/persistNodeValue/calculateDependantNodes/calculateDependantNodes.ts

```diff
@@ -1,24 +1,35 @@
 import { Row, VariableCache } from '@meta/assessment'
 import { NodeUpdates } from '@meta/data'
 
+import { getDependants, isODPCell } from '@server/controller/cycleData/persistNodeValue/getDependants'
```

Comment:
if getDependants, isODPCell are used only here, move the file in this folder, and use relative import (relative import also for Props line below :P )

Reply:
👍 
moved to under utils/